### PR TITLE
renames Function Hooks to "Database Webhooks"

### PR DESCRIPTION
**Pull request by kiwicopple**
cc @joshenlim - we might just need to double-check that nothing is broken in Studio ...

### CODE REVIEW

File: apps/www/_blog/2021-10-28-supabase-series-a.mdx [hunk 1]
```diff
@@ -227,7 +227,7 @@ existing functionality.
 
 - **Row Level Security:** restrict user access to data and files using Postgres Policies.
 - **Realtime data streams:** subscribe to database changes using the logical replication stream.
-- **Function Hooks:** trigger external systems using Postgres triggers and our [async request extension](https://github.com/supabase/pg_net/).
+- **Database Webhooks (previosuly "Function Hooks"):** trigger external systems using Postgres triggers and our [async request extension](https://github.com/supabase/pg_net/).
```
Comment (dannykng) [inline apps/www/_blog/2021-10-28-supabase-series-a.mdx:230]:
```suggestion
- **Database Webhooks (previously called "Function Hooks"):** trigger external systems using Postgres triggers and our [async request extension](https://github.com/supabase/pg_net/).
```

File: apps/www/pages/partners/integrations/index.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export async function getStaticProps() {
     props: {
       partners,
     },
-    // TODO: consider using Next.js' On-demand Revalidation with Supabase function hooks instead
+    // TODO: consider using Next.js' On-demand Revalidation with Supabase database webhooks instead
```
Comment (dannykng) [inline apps/www/pages/partners/integrations/index.tsx:27]: supa small nit:
```suggestion
    // TODO: consider using Next.js' On-demand Revalidation with Supabase Database Webhooks instead
```

File: apps/www/_blog/2021-07-30-supabase-functions-updates.mdx [hunk 3]
```diff
@@ -120,14 +122,14 @@ type DeletePayload = {
 
 ## Hooks technical design: `pg_net v0.1`
 
-As with most of the Supabase platform, we leverage PostgreSQL's native functionality to implement Function Hooks.
+As with most of the Supabase platform, we leverage PostgreSQL's native functionality to implement Database Webhooks (previosuly called "Function Hooks").
```
Comment (dannykng) [inline apps/www/_blog/2021-07-30-supabase-functions-updates.mdx:125]:
```suggestion
As with most of the Supabase platform, we leverage PostgreSQL's native functionality to implement Database Webhooks (previously called "Function Hooks").
```

File: apps/www/_blog/2021-07-30-supabase-functions-updates.mdx [hunk 4]
```diff
@@ -60,13 +60,15 @@ This is especially true for our Enterprise customers, but also Jamstack develope
...
+- _Stage 1:_ Give developers the ability to trigger external HTTP functions - today, using Database Webhooks.
 - _Stage 2:_ Give developers the ability to trigger their own Serverless functions on AWS and GCP - Q4 2021.
 - _Stage 3:_ Release our own Serverless Functions (Supabase Functions) - Q4 for Early Preview customers.
 
 ![Supabase Functions timeline](/images/blog/functions-updates/functions-timeline.png)
 
-## Function Hooks (Alpha)
+## Database Webhooks (Alpha)
+
+(Note: Database Webhooks were previosuly named "Function Hooks")
```
Comment (dannykng) [inline apps/www/_blog/2021-07-30-supabase-functions-updates.mdx:71]:
```suggestion
(Note: Database Webhooks were previously called "Function Hooks")
```

**Review by dannykng — COMMENTED** [review]
Left a few comments!

**Review by joshenlim — APPROVED** [review]
yup studio side looks good! changes are purely UI

**Pull request merged**